# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/moz.build
# Commit: e3051b96067a
# Full Hash: e3051b96067a480b020bd8c984a54491665c765e
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-11-11 15:25:44
# Regressor Bug: 1739448
# File Overlap Count: 1
# Description:
#   Bug 1739448 - Implement a prototype WebGL-backed Canvas2D. r=gfx-reviewers,nical,jrmuizel
#   
#   This mainly provides DrawTargetWebgl, which implements the subset of the DrawTarget
#   API necessary for integration with CanvasRenderingContext2D. It translates them to
#   suitable commands for its internal ClientWebGLContext, which then manages remoting
# ==============================================================================

diff -r 66c30950518d -r e3051b96067a dom/canvas/moz.build
--- a/dom/canvas/moz.build	Thu Nov 11 07:16:57 2021 +0000
+++ b/dom/canvas/moz.build	Thu Nov 11 07:16:58 2021 +0000
@@ -71,6 +71,10 @@
 ]
 # XRWebGLLayer.h must be exported for use by the generated WebXRBinding.h
 
+EXPORTS.mozilla.gfx += [
+    "DrawTargetWebgl.h",
+]
+
 # Canvas 2D and common sources
 UNIFIED_SOURCES += [
     "CanvasGradient.cpp",
@@ -88,6 +92,7 @@
 ]
 
 SOURCES += [
+    "DrawTargetWebgl.cpp",  # Isolate Skia
     "ImageUtils.cpp",
 ]
 